generator/lib/behavior/nestedset/NestedSetBehaviorPeerBuilderModifier.php

```diff
@@ -85,6 +85,13 @@ public function staticAttributes($builder)
  */
 const SCOPE_COL = '" . $tableName . '.' . $this->getColumnConstant('scope_column') . "';
 ";
+        }else{
```

Comment:
Class constants should be available always. It is hard to use them if it is unclear whether they exist or not.